libs/langchain/langchain/document_loaders/s3_directory.py

```diff
@@ -82,7 +86,11 @@ def __init__(
             object is set on the session, the config object used when creating
             the client will be the result of calling ``merge()`` on the
             default config with the config provided to this call.
+
+        :param delete_processed_documents: delete documents after processing
```

Comment:
There could be other strategies like moving to another bucket ?

Reply:
good point. fixed